packages/block-library/src/latest-posts/constants.js

```diff
@@ -1,3 +1,4 @@
 export const MIN_EXCERPT_LENGTH = 10;
 export const MAX_EXCERPT_LENGTH = 100;
 export const MAX_POSTS_COLUMNS = 6;
+export const DEFAULT_EXCERPT_LENGTH = 10;
```

Comment:
```suggestion
export const DEFAULT_EXCERPT_LENGTH = 55;
```

This value must be [a default value defined in block.json](https://github.com/WordPress/gutenberg/blob/129587828d6f2908ddf7e1027ecb14f115e095e0/packages/block-library/src/latest-posts/block.json#L34), otherwise it will not reset properly.